core/src/main/scala/kafka/server/DynamicConfig.scala

```diff
@@ -103,7 +103,7 @@ object DynamicConfig {
       .define(ProducerByteRateOverrideProp, LONG, DefaultProducerOverride, MEDIUM, ProducerOverrideDoc)
       .define(ConsumerByteRateOverrideProp, LONG, DefaultConsumerOverride, MEDIUM, ConsumerOverrideDoc)
       .define(RequestPercentageOverrideProp, DOUBLE, DefaultRequestOverride, MEDIUM, RequestOverrideDoc)
-      .define(ControllerMutationOverrideProp, LONG, DefaultConsumerOverride, MEDIUM, ControllerMutationOverrideDoc)
+      .define(ControllerMutationOverrideProp, DOUBLE, DefaultConsumerOverride, MEDIUM, ControllerMutationOverrideDoc)
```

Comment:
why not changing the type of default value from ```Long``` to ```Double```?

Reply:
good point. i have changed this to use `Int.MaxValue.toDouble` by default like we do for the request quota.